themes/default/content/docs/clouds/aws/get-started/_index.md

```diff
@@ -12,8 +12,1418 @@ menu:
 aliases:
 - /docs/quickstart/aws/
 - /docs/get-started/aws/
+- /docs/quickstart/aws/begin/
+- /docs/quickstart/aws/install-pulumi/
+- /docs/quickstart/aws/install-language-runtime/
+- /docs/quickstart/aws/configure/
+- /docs/get-started/aws/install-pulumi/
+- /docs/get-started/aws/install-language-runtime/
+- /docs/get-started/aws/configure/
+- /docs/get-started/aws/begin/
+- /docs/quickstart/aws/create-project/
+- /docs/get-started/aws/create-project/
+- /docs/quickstart/aws/deploy-changes/
+- /docs/get-started/aws/deploy-changes/
+- /docs/quickstart/aws/deploy-stack/
+- /docs/get-started/aws/deploy-stack/
+- /docs/quickstart/aws/destroy-stack/
+- /docs/get-started/aws/destroy-stack/
+- /docs/quickstart/aws/modify-program/
+- /docs/get-started/aws/modify-program/
+- /docs/quickstart/aws/next-steps/
+- /docs/get-started/aws/next-steps/
+- /docs/quickstart/aws/review-project/
+- /docs/get-started/aws/review-project/
 ---
 
-{{< cloud-intro "AWS" >}}
+Pulumi's infrastructure-as-code SDK helps you create, deploy, and manage AWS containers, serverless functions, and infrastructure using familiar programming languages.
 
-{{< get-started-stepper >}}
+This tutorial takes you through the following steps to easily deploy a static website.
+
+## Install Pulumi
+
+{{< install-pulumi >}}
+{{% notes "info" %}}
+All Windows examples in this tutorial assume you are running in PowerShell.
+{{% /notes %}}
+{{< /install-pulumi >}}
+
+### Install language runtime
+
+{{< chooser language "javascript,typescript,python,go,csharp,java,yaml" >}}
+
+{{% choosable language "javascript,typescript" %}}
+
+Install [Node.js](https://nodejs.org/en/download/) or [Install Node.js via package manager](https://nodejs.org/en/download/package-manager/).
+{{% /choosable %}}
+
+{{% choosable language python %}}
+{{< install-python >}}
+{{% /choosable %}}
+
+{{% choosable language go %}}
+{{< install-go >}}
+{{% /choosable %}}
+
+{{% choosable language "csharp,fsharp,visualbasic" %}}
+{{< install-dotnet >}}
+{{% /choosable %}}
+
+{{% choosable language "java" %}}
+{{< install-java >}}
+{{% /choosable %}}
+
+{{% choosable language "yaml" %}}
+{{< install-yaml >}}
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+### Configure Pulumi to access AWS account
```

Comment:
Suggest either "Configure Pulumi to access your AWS account" or "Configure Pulumi to access AWS". (Sounds a bit robotic otherwise.) Looks like the previous version had "your", so suggest we stick with that:

```suggestion
### Configure Pulumi to access your AWS account
```